src/admin/components/UserResumeTable/index.tsx

```diff
@@ -79,11 +110,28 @@ const UserResumeTable: React.FC<UserResumeTableProps> = (props) => {
     {
       title: 'Planned Graduation',
       dataIndex: 'user.graduationYear',
+
+      filterMultiple: true,
+      filters: yearFilters, // [{ text: "2016", value: "2016" }],
+
+      onFilterDropdownVisibleChange: (visible) => {
+        if (visible) {
+          setTimeout(() => updateYearFilters());
+        }
+      },
+
+      // Value we are filtering by, record = content of current row
+      onFilter: (value: string, record: UserResume) => `${record.user.graduationYear}` === value,
```

Comment:
Since the graduation year type is a number, does it make sense to check string equality here?

We probably want to check numeric equality of `record.user.graduationYear === value`, however looking at this line of code above,
```
filters: yearFilters, // [{ text: "2016", value: "2016" }],
```
we can't do numeric equality unless we set value to be the **number** 2016 rather than the string. 

yearFilters should be something like 
```
[{
    "text": "2016",
    "value": 2016
}]
```

Reply:
I believe ColumnProps only takes a string. I get errors when trying to make it a number. If this is not the case please let me know and I'll spend some time on it.

<img width="422" alt="image" src="https://user-images.githubusercontent.com/54158686/229886031-844f2e40-1c51-433b-b111-b3ca7d53f460.png">

Reply:
```suggestion
      onFilter: (value: string, record: UserResume) => record.user.graduationYear === value,
```

I explained the small tweak we need above in a different comment for the value key in setYearFilters. We don't need to modify the columns in any way for this filter check. If you can get the previous line of code working correctly, this should also work in a cleaner way.